nfl/react-metrics
# refactor(build): remove polyfill and add umd build

#19 

manual test instruction:

- `npm test`
- `npm install && npm run examples` on host machine
- open http://10.0.2.2:8080 and check the examples work in IE 10 on Windows VM.
- `npm run build:umd && npm run build:umd:min` and verify the umd file in `dist` folder.
